.gitignore

```diff
@@ -6,7 +6,7 @@ lib-cov
 *.out
 *.pid
 *.gz
-
+ve/
```

Comment:
Since this is not a Python project, so we probably don't need `ve/` in `.gitignore`.